guess-the-country/country-backend/country/api.py

```diff
@@ -92,10 +92,11 @@ def generate_random(polygon):
 
 @api.post("/predict")
 def predict(file: UploadFile = File(...)):
-    image = load_image(file)
+    encoded_data = file.file.read()
+    image = load_image(encoded_data)
     pre_image = preprocess(image)
     prediction_id = uuid.uuid4()
-    label = predict_image(pre_image, model)
+    label = predict_image(pre_image, country_array)
```

Comment:
Den Teil bitte in `model.predict` verschieben, so dass man das einzeln testen kann.

`api.py` sollte möglichst nur die Anfragen entgegennehmen und weiterreichen, aber selbst möglichst wenig "Fachlogik" enthalten.